### ISSUE TITLE
Strapi

### ISSUE THREAD

**Issue by KaustubhTrivedi** [javascript-shinobis/nike-clone-app#15]
How are you guys planning to host strapi? I am interested in working on this project and I have professional experience in working with strapi.

**Comment by abhimishra01** [comment]
> How are you guys planning to host strapi? I am interested in working on this project and I have professional experience in working with strapi. I recently delivered a strapi project in my company

yes that's the plan in our TODO pipeline, we currently have 3 active collaborators, you are welcome to collaborate with us actively

if you are available we all can get our one of weekly sync calls and plan the tasks together?
let me know which schedule works for you

**Comment by KaustubhTrivedi** [comment]
Yes I am available. I saw that you guys are using app directory and strapi so I got pumped up seeing this😂😂.

**Comment by abhimishra01** [comment]
> Yes I am available. I saw that you guys are using app directory and strapi so I got pumped up seeing this😂😂.

will 8pm 22nd June 2023 IST work ?
we all can meet [here](https://meet.google.com/xyz-ihef-mgx)

**Comment by KaustubhTrivedi** [comment]
I'll join the call if I am available

**Comment by abhimishra01** [comment]
> I'll join the call if I am available

Hi, let me know if you are available at 8

**Comment by KaustubhTrivedi** [comment]
I will be available👍

**Comment by bhanu0187** [comment]
Hi @KaustubhTrivedi will you be able to join today?
